x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -184,25 +182,6 @@ const StatefulEventsViewerComponent: React.FC<Props> = ({
     [dispatch, id]
   );
 
-  const refetchQuery = (newQueries: inputsModel.GlobalQuery[]) => {
-    newQueries.forEach((q) => q.refetch && (q.refetch as inputsModel.Refetch)());
-  };
-
-  const addToCaseBulkActions = useBulkAddToCaseActions();
-  const bulkActions = useMemo(
```

Comment:
Bulk Actions state has been moved up the tree so that it is easier for implementing function to add their own bulk actions.